### PR TITLE
Issue#1878: Upgrade the org.jacoco:jacoco-maven-plugin

### PR DESCRIPTION
**Pull request by tanvimehta**
Upgraded dependency version. Ran all tests successfully. https://github.com/strongbox/strongbox/pull/1880 depends on this pull request.

### CODE REVIEW

**Comment by carlspring** [comment]
Hi @tanvimehta ,

Thanks! This looks good, but I would like to request the change above, so that we can run the full build on our CI server.

**Comment by carlspring** [comment]
Thank you very much! :)

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-ISSUE-1878-SNAPSHOT</version>
```
Comment (steve-todorov) [inline pom.xml:9]:
```suggestion
    <version>1.0-issue-1878-dep-upgrade-SNAPSHOT</version>
```

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1878-dep-upgrade-SNAPSHOT</version>
```
Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```

Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```

The tests have passed in our CI. I am now reverting this change and will be closing the other (test) pull request. Thank you very much! :)

**Pull request merged**